src/modules/Nis.rb

```diff
@@ -730,6 +733,8 @@ def Import(settings)
     # Dump the NIS settings to a map, for autoinstallation use.
     # @return $["start":, "servers":[...], "domain":]
     def Export
+      return {} unless Yast::Package.Installed(NIS_CLIENT_PACKAGE)
```

Comment:
Do not have consequences when Import and Export? I was wondering if could affect AutoYaST config for example.

Reply:
That's the idea. Do not export anything when the package is not installed.

Reply:
Oh, now I get it. You are right, I have fixed the problem.